### ISSUE TITLE
Render sass with options passed on the magnet config

### ISSUE THREAD

**Issue by zenorocha** [mthadley/magnet-plugin-sass#1]
For example:

```
pluginsConfig: {
      sass: {
        src: 'static/styles/main.scss',
        includePaths: ['node_modules'],
        outputStyle: 'compressed'
      }
    }
```

**Comment by mthadley** [comment]
@zenorocha Totally makes sense. 👍  🎉

**Comment by mthadley** [comment]
@zenorocha Released `v2.1.0`, see if that helps.

**Comment by zenorocha** [comment]
Works perfectly, thanks!
